packages/samples/cloudant/javascript/create-document.js

```diff
@@ -0,0 +1,56 @@
+/**
+  *
+  * main() will be invoked when you Run This Action with:
+  *
+  * wsk action invoke create-cloudant-document --blocking
+  *
+  * @param Whisk actions accept a single parameter,
+  *        which must be a JSON object.
+  *
+  * In this case, the params variable will look like:
+  *     {
+  *            "cloudant_package": "xxxx",
+  *            "doc": "xxxx",
+  *            "namespace": "xxxx"
+  *        }
+  *
+  * @return which must be a JSON object.
+  *         It will be the output of this action.
+  *
+  */
+
+var openwhisk = require("openwhisk");
+
+function main(params) {
+
+    var wsk = openwhisk();
+
+    // read cloudant package name
+    var cloudant = params['cloudant_package'];
+
+    // read document
+    var doc = params['doc'];
+
+    var namespace = params['namespace'];
+
+    // access namespace as enviornment variables if not specified in params
+    if (typeof namespace === 'undefined' || namespace === null) {
+        namespace = process.env['__OW_NAMESPACE'];
+    }
+
+    var cloudantActionName = "/" + namespace + "/" + cloudant + "/create-document";
+
+       return wsk.actions.invoke({
```

Comment:
Thank you for the clarification.

For node actions, it's arguably more convenient to then use the `npm cloudant` sdk rather than invoke an action to do the same. The actions exist as a wrapper around that module so that one can compose them more easily into sequences for example.

That said, invoking the cloudant action vs another action from a whisk perspective is the same.